src/flownet/ahm/_ahm_iteration_analytics.py

```diff
@@ -381,10 +390,11 @@ def save_iteration_analytics():
     metrics = list(args.metrics.replace("[", "").replace("]", "").split(","))
 
     # Load ensemble of FlowNet
-    (df_sim, realizations_dict, iteration,) = make_dataframe_simulation_data(
+    (df_sim, iteration, nb_real) = make_dataframe_simulation_data(
         args.runpath,
         args.eclbase,
         list(args.quantity.replace("[", "").replace("]", "").split(",")),
+        np.datetime64(args.end),
```

Comment:
Why work with `np.datetime64` and not just with standard python dates?

Reply:
this type is used because that's the type for the dates when loading data from the summary file, and in order to filter the created datasets the dates type must be the same